--- gitweb.cgi.orig
+++ gitweb.cgi
@@ -3125,7 +3125,7 @@
 				# only directories can be git repositories
 				return unless (-d $_);
 				# need search permission
-				return unless (-x $_);
+				#return unless (-x $_);
 				# don't traverse too deep (Find is super slow on os x)
 				# $project_maxdepth excludes depth of $projectroot
 				if (($File::Find::name =~ tr!/!!) - $pfxdepth > $project_maxdepth) {
