--- config.py.orig
+++ config.py
@@ -184,7 +184,7 @@
                     if sections and section not in sections:
                         continue
                     v = self.get(section, item) + b"\n" + m.group(1)
-                    self.set(section, item, v, b"%s:%d" % (src, line))
+                    self.set(section, item, v, ("%s:%d" % (src, line)).encode())
                     continue
                 item = None
                 cont = False
@@ -218,7 +218,7 @@
                 cont = True
                 if sections and section not in sections:
                     continue
-                self.set(section, item, m.group(2), b"%s:%d" % (src, line))
+                self.set(section, item, m.group(2), ("%s:%d" % (src, line) ).encode() )
                 continue
             m = unsetre.match(l)
             if m:
